bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
```

Comment:
I think that if we're adding new linter bears we should add all the configs from now on ... The current system is causing double work where someone just writes boilerplate code to get it into the CI and there is another issue to add configs to it ...
Our linter stuff is sufficiently simple that they can be both added to the same issue